extensions/ui/org.eclipse.smarthome.ui.basic/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+  <config-description uri="ui:basic">
+	<parameter name="iconType" type="text" required="true">
+	    <label>Icon Format</label>
+	    <description>Defines whether bitmap or vector icons are used by the UI.</description>
+	    <options>
+	       <option value="png">Bitmap</option>
```

Comment:
Indeed - just applied the formatter, now all are tabs (is this something we want to address in future?)